update examples/README.md with more examples and instructions fix (#290)
hexops/vecty: example/README.md

Other compilers such as [GopherJS](https://github.com/gopherjs) may work so long as they are compliant with the official Go 1.14+ compiler (support modules, the `syscall/js` interface, reflection, etc.)

Vecty currently can only be built to run in web browsers.

## More examples

> Github user soypat has [created a repo call vecty-examples](https://github.com/soypat/vecty-examples) with examples such as a snake game and a user interface using [Material Design CSS library](https://material.io/).

Reviewer: ```suggestion

* [soypat's vect-examples](https://github.com/soypat/vecty-examples), including a snake game and an example of using the [Material Design CSS library](https://material.io/)
```
